packages/manager/modules/dbaas-logs/src/logs/order/order.component.js

```diff
@@ -0,0 +1,13 @@
+import controller from './logs-order.controller';
+import template from './logs-order.html';
+
+export default {
+  bindings: {
+    me: '<',
+    catalog: '<',
+    formattedPrice: '<',
+  },
+  controller,
+  controllerAs: 'ctrl',
```

Comment:
If it's a new component, we prefer to use the default value (`$ctrl`)

Reply:
i've replace all :)